src/components/screens/ExerciseInstruction.stories.tsx

```diff
@@ -0,0 +1,17 @@
+// @ts-ignore TODO: add declaration file
+import { storiesOf } from '@storybook/react-native';
+import * as React from 'react';
+
+// @ts-ignore TODO: add declaration file
+import getExercise from '../exercises';
+import { ExerciseInstruction } from './ExerciseInstruction';
+
+storiesOf('screens/ExerciseInstruction', module).add('FindLetter', () => {
+  const FindLetter = getExercise('FindLetter');
+
+  const exercise = new FindLetter.Exercise(FindLetter.fixtures[0].props);
+
+  return (
+    <ExerciseInstruction exercise={exercise} Component={FindLetter.Component} />
```

Comment:
I like this, but don't quite understand: Is the component `ExerciseInstruction` supposed to work with any Component? If thats the case, thats great! Then adding it to the course shouldn't be too difficult, we could even think about just including a help button, where the animation is played for the current exercise?

Reply:
Probably not, no. I'm leaning more to making an instruction for each exercise (since it is super dependent on the view and therefore the buttons etc. one has). But we can probably extract some common logic and connect the Instruction to the Exercise Type (so that we can do the help button thingy you spoke of). But what works: the animation works for arbitrary inputs (as soon as `animation` is calculated from props). With a little change we could probably also animate "wrong" inputs if we need that later.